src/Common/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -1279,6 +1283,14 @@ private void ThrowIfOperationInProgress(Task operationTask, [CallerMemberName] s
                 throw new InvalidOperationException(SR.Format(SR.net_Websockets_AlreadyOneOutstandingOperation, methodName));
             }
         }
+        
+        private void ThrowIfDisposed()
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(GetType().FullName);
```

Comment:
Alternatively, since this is a sealed class... use nameof(ManagedWebSocket) and you can avoid reflection altogether.